### ISSUE TITLE
Extreme Entity Crusher - swords interaction

### ISSUE THREAD

**Issue by Slendooo** [GTNewHorizons/GT-New-Horizons-Modpack#15490]
### Your GTNH Discord Username

Slendooo

### Your Pack Version

2.5.1

### Your Proposal

There should be an option to keep swords inside input bus of Extreme Entity Crusher while their effects are still applied as it was in version before, now every sword is pulled automatically from input bus to new slot in the controller thus you cant interact with it.

### Your Goal

Swords without infinite durabillity were made by this change not worth to use in this machine because you cant automatically output them and repair them, also tinkers construct weapons are not even allowed inside of it so there isnt even much to choose from. 

### Your Vision

If extreme extermination chamber is supposed to be an alternative to classic mobtraps with spawners because it causes less lag, then this change with pulling sword into the controller is kinda contradicting this idea, it is forcing people who would want to automate, for example wither skulls with skullfire sword back to spawners, back to killer joe, because you are actually able to pull the sword out and repair it.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Hikari1nVoid** [comment]
Use diamond spikes.
Also even with 16A UXV I can still managed to get my arthana back.Durability drain is not that fast.In your case skullfire has 1561 durability and it will have plenty of time to take it out b4 break.

**Comment by Slendooo** [comment]
> Use diamond spikes. Also even with 16A UXV I can still managed to get my arthana back.Durability drain is not that fast.In your case skullfire has 1561 durability and it will have plenty of time to take it out b4 break.

Thats true, that the durabillity drain isnt huge, but what i loved about this machine was that i could fully automate it so i never had to think about it at all, it did its thing and i was not needed to interact with it at all after it was built. Now i cant forget to check the sword from time to time bcs otherwise it will break and thats kinda annoying me (even more so if i will build more of them in future). 

And afterall this pack is mostly about automation so not being able to fully automate this seems kinda out of place.

**Comment by kuba6000** [comment]
What about an option in GUI to output the sword into the output bus when it is at 1 durability?
